src/mistral_common/protocol/instruct/normalize.py

```diff
@@ -130,7 +131,7 @@ def _normalize_tool_call(self, tool_call: ToolCall) -> ToolCall:
         )
 
     def _aggregate_assistant_messages(self, messages: List[UATS]) -> AssistantMessageType:
-        aggregated_content: List[str] = []
+        aggregated_content: List[Union[TextChunk, ThinkChunk]] = []
```

Comment:
Think we should still allow `str` no?

Reply:
It's still supported changed the name of variables to better reflect that